src/internal/builtInServices.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CallServiceResult } from '@fluencelabs/avm';
+import { encode, decode } from 'bs58';
+import { PeerIdB58 } from 'src';
+import { GenericCallServiceHandler, ResultCodes } from './commonTypes';
+import { KeyPair } from './KeyPair';
+
+const success = (result: any): CallServiceResult => {
+    return {
+        result: result,
+        retCode: ResultCodes.success,
+    };
+};
+
+const error = (error: string): CallServiceResult => {
+    return {
+        result: error,
+        retCode: ResultCodes.unknownError,
```

Comment:
Why is it always `ResultCodes.unknownError`?

Reply:
Seems like error code = 1 works fine. Do you have any ideas how to improve it?

Reply:
I just don't understand the meaning yet. You throw e.g.,
```js
error(`All arguments of 'concat' must be arrays: arguments ${str} are not`);
```
and it's an unknown error?